README.md

```diff
@@ -286,9 +287,14 @@ Now, let's edit our submit file to properly handle these new arguments and outpu
 	queue 1
 
 
-Notice the added `arguments = input.txt` information. The `arguements` option specifies what arguments should be passed to the executable. 
+Notice the added `arguments = input.txt` information. The `arguments` option specifies what arguments should be passed to the executable. 
 
-The `transfer_input_files` and `transfer_output_files` options need to be included as well.  When jobs are deployed on the Open Science Grid, they are sent only with files that are specified. Additionally, only the specified output files are returned with the job. **Any output not transferred back, with the exception of our `error`, `output`, and `log` files, are discarded at the end of the job.**
+Due to the distributed configuration of the OSG this job will need to bring along a copy of `input.txt` from the 
+login node, where the job is submitted, to the execute node where the job will run by using `transfer_input_files`. 
```

Comment:
This sentence is a bit ambiguous, and might be fixed by using 'transfer_input_files' at the beginning of the sentence.

Reply:
agreed! I have revised the text accordingly